src/Service/ResourceService.php

```diff
@@ -15,6 +15,7 @@ public function __construct(
         private readonly CacheInterface $resourceCache,
         private readonly SerializerInterface $serializer,
         private readonly MetricsHelper $metricsHelper,
+        private readonly array $excludesResources,
```

Comment:
Unfortunately i didn't notice this typo previously

```suggestion
        private readonly array $excludedResources,
```